CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 - Swift 5 & Xcode 10.2 migration (#45, kudos to @janmisar)
 - base classes for VCs, VMs, FCs (#45, kudos to @janmisar)
 
+## 6.0.4
```

Comment:
How do you know it is going to be 6.0.4? 🤔

Reply:
I'm a magician 🎩 No, I just mechanically followed the changelog and, well, made a mistake, will fix that!